app/src/main/java/io/appium/settings/RecorderService.java

```diff
@@ -0,0 +1,135 @@
+package io.appium.settings;
```

Comment:
Please add the standard header to all newly added files

Reply:
done